prod/westeurope/internal/api/storage_user-data-download/management_policy/terragrunt.hcl

```
@@ -0,0 +1,36 @@
+# Internal
+dependency "storage_account" {
+  config_path = "../account"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_storage_management_policy?ref=v3.0.3"
+}
+
+inputs = {
+  storage_account_id = dependency.storage_account.outputs.id
+
+  rules = [
+    {
+      name    = "deleteafter14days"
+      enabled = true
+      filters = {
+        prefix_match = []
```

```suggestion
        prefix_match = ["user-data-download"]
```
add prefix to apply policy only on `user-data-download` container 
cfr. https://github.com/pagopa/io-infrastructure-live-new/blob/master/prod/westeurope/operations/storage_account_logs/management_policy/terragrunt.hcl#L23

---

I think no other kind of blob is stored here, but fine for me

---

we should reference this I think: https://github.com/pagopa/io-infrastructure-live-new/blob/adb173ef3846238cd0936bad04aab416e27cc819/prod/westeurope/internal/api/storage_user-data-download/container_user-data-download/terragrunt.hcl#L15

---

no other kind of blob now is stored here but we need to stay safe in case an unwanted change ;)
yeah u can link with dependency

---

done in https://github.com/pagopa/io-infrastructure-live-new/pull/637/commits/51eec7db1190e1663dfeebd3c6cfbab3d405664d

---

Have u applied the change?

---

nope